solr/solrj/src/test/org/apache/solr/client/solrj/impl/Http2SolrClientTest.java

```diff
@@ -661,12 +661,13 @@ private void verifyServletState(Http2SolrClient client, SolrRequest<?> request)
   public void testQueryString() throws Exception {
 
     final String clientUrl = jetty.getBaseUrl().toString() + "/debug/foo";
-    try (Http2SolrClient client = getHttp2SolrClient(clientUrl)) {
+    UpdateRequest req = new UpdateRequest();
+    setReqParamsOf(req, "serverOnly", "notServer");
```

Comment:
it doesn't matter but its clearer to place this after client creation instead of before.  This is consistent with tests below.